### PR TITLE
Generate chain specs and genesis for dev and stage

### LINKED ISSUE #161: Create Genesis for all dev envs
**Endowed accounts list:**
- Zero treasury: {to be updated}
- GameDAO treasury: {to be updated}
- Game3 foundation treasury: {to be updated}
- Faucet bot: 3YGofbXojRXSJrHT4pRxdcmvYZ2Tbwux9wSjv2gwVDTV3oAM
- Sudo:
   - DEV: {to be updated}
   - STAGE: {to be updated}
   
Token distribution:
- Faucet bot:
   - ZERO: {to be updated}
   - GAME:  {to be updated}
   - PLAY: {to be updated}
- Sudo:
   - ZERO: {to be updated}
   - GAME:  {to be updated}
   - PLAY: {to be updated}

Token distribution rules for treasury accounts defined in figma: https://www.figma.com/file/6BD2pkVrBqO0GyJTvp1t5r/Governance-%26-Treasury-Setup?node-id=0%3A1

**Steps:**
- Prepare functions for chainspec generation
- Generate parachain chainspec for DEV -> image public.ecr.aws/zero.io/subzero:{version}-dev updated
- Generate relaychain chainspec for DEV -> public.ecr.aws/zero.io/rococo:{version}-dev updated
- Generate  chainspec for STAGE -> image public.ecr.aws/zero.io/subzero:{version}-stage updated
- Generate relaychain chainspec for STAGE -> public.ecr.aws/zero.io/rococo:{version}-stage updated
- DEV and STAGE deploy changes
- Test if all accounts have expected balances

### PR DESCRIPTION
**Pull request by vovacha**
Genesis (Subzero runtime):
- Balances
- Tokens
- Control

chain_spec (Subzero runtime):
- DEV
- STAGE

Test scenario:
- Check if all endowed accounts have expected balances
- Check if all orgs were created with proper treasury addresses
- Check if all collators have proper keys
- Build ok
- Test ok

**!Rebase and merge!**
Commit was used to tag a docker image.

### CODE REVIEW

File: .docker/subzero.Dockerfile [hunk 1]
```diff
@@ -44,14 +44,15 @@ RUN mv /usr/share/ca* /tmp && \
...
+COPY --from=builder /subzero/.docker/chainspec /chainspec
 
 # checks
 RUN ldd /usr/local/bin/subzero && \
 	/usr/local/bin/subzero --version
 
 # Shrinking
-RUN rm -rf /usr/lib/python* && \
-	rm -rf /usr/bin /usr/sbin /usr/share/man
+# RUN rm -rf /usr/lib/python* && \
```
Comment (vayesy) [inline .docker/subzero.Dockerfile:54]: I am wondering what was the reason to execute these lines, @2075?
It doesn't seem to reduce image size significantly.

File: bin/subzero/node/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "subzero-node"
-version = "3.2.62"
+version = "3.2.63"
```
Comment (vayesy) [inline bin/subzero/node/Cargo.toml:3]: Update runtime cargo version as well.

**Review by vayesy — COMMENTED** [review]
As far as I remember we decided to keep runtime versions for all envs the same - zero, subzero, standalone. Is this still the case? If so, other chain versions apart from subzero should also be updated to 63.

File: .docker/subzero.Dockerfile (hunk 1 above)
Comment (2075) [reply .docker/subzero.Dockerfile:54]: efficiency and less clutter on the image

**Comment by 2075** [comment]
just focus on getting one done

File: .docker/subzero.Dockerfile (hunk 1 above)
Comment (vovacha) [reply .docker/subzero.Dockerfile:54]: It removes shell interpreters and I'm unable to run entrypoint.sh, that's why I removed it. Also, there is zero to little impact on image size.

Comment (2075) [reply .docker/subzero.Dockerfile:54]: it is removing non required entrypoints, feel free to add what you need but a node image should be single minded, no?

**Review by 2075 — APPROVED** [review]

File: bin/subzero/node/Cargo.toml (hunk 2 above)
Comment (vovacha) [reply bin/subzero/node/Cargo.toml:3]: Why we should increment the version for Zero even if there were no changes?
As for standalone, I still don't see any reason of supporting it at all.

**Pull request merged**